models/balancer/balances/balancer_token_balance_changes.sql

```diff
@@ -0,0 +1,46 @@
+{{ config(
+    schema = 'balancer',
+    alias = 'token_balance_changes',
+    post_hook='{{ expose_spells(\'["arbitrum", "avalanche_c", "base", "ethereum", "gnosis", "optimism", "polygon", "zkevm"]\',
+                                "project",
+                                "balancer",
+                                \'["viniabussafi"]\') }}'
```

Comment:
We may apply the new format for all post_hook in this PR:
```suggestion
    post_hook='{{ expose_spells(blockchains = \'["arbitrum", "avalanche_c", "base", "ethereum", "gnosis", "optimism", "polygon", "zkevm"]\',
                                spell_type = "project",
                                spell_name = "balancer",
                                contributors = \'["viniabussafi"]\') }}'
```

Reply:
will work on it!

Reply:
done! also took the chance to remove some post hooks from chain-specific spells in order to clean up the explorer